lib/iob/calculate.js

```diff
@@ -16,11 +16,12 @@ function iobCalcExponential(treatment, time, dia, peak, profile) {
 
     var end = dia * 60;  // end of insulin activity, in minutes
  
+    // Use custom peak time if values are valid
     if (profile.useCustomPeakTime && profile.insulinPeakTime !== undefined) {
-        if (profile.insulinPeakTime < 35 || profile.insulinPeakTime > 120) {
-            console.error('Insulin Peak Time is only supported for values between 35 to 120 minutes');
-        } else {
+        if (profile.insulinPeakTime > 35 && profile.insulinPeakTime < 120) {
```

Comment:
You've effectively changed the minimum allowed value from 35 to 36.  I doubt anyone is using it, but probably better for consistency to make this a >= 35?

Reply:
Good point. Should do `<= 120` for consistency, too.